sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/api/Azure.MixedReality.ObjectAnchors.Conversion.netstandard2.0.cs

```diff
@@ -147,37 +173,13 @@ public static partial class ObjectAnchorsConversionModelFactory
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         public override int GetHashCode() { throw null; }
     }
-}
-namespace Azure.MixedReality.ObjectAnchors.Conversion.Models
-{
-    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
-    public readonly partial struct ConversionErrorCode : System.IEquatable<Azure.MixedReality.ObjectAnchors.Conversion.Models.ConversionErrorCode>
+    public partial class UnsupportedAssetFileTypeException : System.Exception, System.Runtime.Serialization.ISerializable
```

Comment:
Should this be `NotSupportedAssetFileTypeException` ? /cc @KrzysztofCwalina @tg-msft

Reply:
Addressed in the meantime

Reply:
Good catch Alex.  .NET does prefer [`NotSupported`](https://docs.microsoft.com/en-us/dotnet/api/?term=notsupported) over [`Unsupported`](https://docs.microsoft.com/en-us/dotnet/api/?term=unsupported).  Although I would not recommend deriving from the existing `NotSupportedException` because [that has different specific use cases](https://docs.microsoft.com/en-us/dotnet/api/system.notsupportedexception?view=net-5.0#throwing-a-notsupportedexception-exception).

The only thing is `NotSupportedAssetFileTypeException` reads a little funny to me.  I think `AssetFileTypeNotSupportedException` sounds better.  What do you think @jastenze?

Reply:
That works for me